numpy/exceptions.py

```diff
@@ -103,7 +103,7 @@ class AxisError(ValueError, IndexError):
     For compatibility with code written against older numpy versions, which
     raised a mixture of `ValueError` and `IndexError` for this situation, this
     exception subclasses both to ensure that ``except ValueError`` and
-    ``except IndexError`` statements continue to catch `AxisError`.
+    ``except IndexError`` statements continue to catch `~exceptions.AxisError`.
```

Comment:
Maybe format as code instead of a link since it is referring to itself?
```suggestion
    ``except IndexError`` statements continue to catch ``AxisError``.
```